.github/workflows/publish.yml

```diff
@@ -381,10 +375,6 @@ jobs:
       release_url: ${{ steps.release.outputs.url }}
 
     steps:
-      - name: Set tag
-        run: |
-          echo "TAG=$(git tag --points-at ${{github.sha}})" >> $GITHUB_ENV
-
```

Comment:
The tag is removed here in favor of the tag we computed in the first step (the first job is the single source of truth whether we should create a release or not)